### DISCUSSION TITLE
An example of a REST API with an email filter in the profile table?

### DISCUSSION THREAD

**Discussion by organicnz** [supabase/supabase#12196] [Questions] [answered]
Hi folks,

I'd like to check/filter if the table `profiles` contains a particular email in the rows. 

Could you pls hepl with the proper GET query on this endpoint? This one returns an error:  
`https://iazmxxxxxxxxx.supabase.co/rest/v1/profile?email=example@email.com'`

Thank you :)

**Comment by steve-chavez** [comment, marked as answer]
Try with

```
https://iazmxxxxxxxxx.supabase.co/rest/v1/profile?email=eq.example@email.com
```

For more details, see https://postgrest.org/en/stable/api.html#horizontal-filtering-rows

**Reply by organicnz** [reply]
Thanks so much, mate. It works now hehe
